src/components/molecules/ChatPoll/ChatPoll.tsx

```diff
@@ -113,7 +115,7 @@ export const ChatPoll: React.FC<ChatPollProps> = ({
           style={{ width: `${question.share}%` }}
         />
         <span className="ChatPoll__text-count">{question.share}%</span>
-        {question.name}
+        <RenderMarkdown text={question.name} />
```

Comment:
pushed de-linting and adding markdown-ification of the polls, so they would look like
![image](https://user-images.githubusercontent.com/39889/120654775-feba1300-c44f-11eb-9968-965de4396fa4.png)

didn't bother about adding emoji picker to polls input entries